tools/realsense-viewer/CMakeLists.txt

```diff
@@ -63,19 +63,18 @@ if(DEFINED OpenCV_DIR  AND  IS_DIRECTORY ${OpenCV_DIR})
     include_directories( ../../wrappers/opencv )
 
 endif()
-if(DEFINED INTEL_OPENVINO_DIR  AND  IS_DIRECTORY ${INTEL_OPENVINO_DIR})
+if(ON)
```

Comment:
This line should not part of the PR

Reply:
I think you can rewrite this part after merging of this PR